apps/mudita-center-e2e/src/specs/help/help-contact-support.ts

```diff
@@ -0,0 +1,150 @@
+    await modalHeader.waitForDisplayed()
+    const modalHeaderTitle = await HelpModalPage.modalHeaderTitle
+    await expect(modalHeaderTitle).toHaveText("Mudita Center Support")
+  })
+
+  it("Check contents of Mudita Help Contact Support", async () => {
+    const emailInput = await HelpModalPage.emailInput
+    await emailInput.waitForDisplayed()
+    await emailInput.waitForEnabled()
+    await expect(emailInput).toHaveAttributeContaining(
+      "placeholder",
+      "Your email"
+    )
+
+    const descriptionInput = await HelpModalPage.descriptionInput
+    await descriptionInput.waitForDisplayed()
+    await descriptionInput.waitForEnabled()
+    await expect(descriptionInput).toHaveAttributeContaining(
+      "placeholder",
+      "How can we help?"
+    )
+
+    const attachmentsList = await HelpModalPage.attachmentsList()
+    await expect(attachmentsList).toHaveLength(1)
+
+    const attachment = await HelpModalPage.singleAttachment
+    const d = new Date()
+    const day = d.getDate()
+    const month = d.getMonth() + 1
+    const year = d.getFullYear()
+    await expect(attachment).toHaveText(
+      `${year}-${(month > 9 ? "" : "0") + month}-${
+        (day > 9 ? "" : "0") + day
+      }.zip`
+    )
+  })
+
+  it("Fill the form correctly", async () => {
+    await testsHelper.insertTextToElement(
+      await HelpModalPage.emailInput,
+      "tomasz.malecki@mudita.com"
+    )
+
+    await testsHelper.insertTextToElement(
+      await HelpModalPage.descriptionInput,
+      "This is test message from automatic tests execution. Please discard it"
+    )
+
+    const sendButton = await HelpModalPage.sendButton
+    await expect(sendButton).toBeClickable()
+    await expect(sendButton).toBeEnabled()
+    await expect(sendButton).toHaveText("SEND")
+
+    console.log(await browser.status())
+    console.log(await browser.getUrl())
+    console.log(await browser.getTitle())
+    console.log(await browser.getWindowHandle())
+    // const helpSupportRequestMock = await browser.mock(
+    //   "https://mudita.freshdesk.com/api/v2/",
+    //   {
+    //     method: "post",
+    //   }
+    // )
+    //console.warn("helpSupportRequestMock.calls.length")
+    //console.warn(helpSupportRequestMock.calls.length)
+
+    //TODO temporary statusCode: 500 to use online but still see respondOverwrite
+    //TODO move to fixtures
+    const strictResponseMock = await browser.mock("**", {
+      // mock all json responses
+    })
+
+    await strictResponseMock.respond([{ statusCode: 500 }])
+    await strictResponseMock.abort("Failed")
+
+    // helpSupportRequestMock.respond(
+    //   {
+    //     id: 23058,
+    //     statusCode: 500,
+    //   },
+    //   {
+    //     statusCode: 500,
+    //     headers: {
+    //       contentType: "application/json; charset=utf-8",
+    //     },
+    //   }
+    // )
+
+    await sendButton.click()
+    await sleep(3000)
+
+    expect(strictResponseMock.calls.length).toBe(1)
+  })
+})
```

Comment:
Are you sure that it should be a part of the PR?

Reply:
Thanks for asking. No thats wrong. I will prepare new branch and PR instead